Assets/Scripts/Control&Input/PlayerMovement.cs

```diff
@@ -11,6 +11,8 @@ enum PlayerState
     DEAD
 }
 
+[RequireComponent(typeof(Rigidbody))]
+[RequireComponent(typeof(Collider))]
```

Comment:
`RequireComponent` allows for up to three components to be required:

```cs
[RequireComponent(typeof(Rigidbody), typeof(Collider))]
public class PlayerMovement : MonoBehaviour {...}
```

Reply:
Thanks, refactored as per your suggestion.